roles/minemeld/tasks/webui.yml

```diff
@@ -5,7 +5,8 @@
     clone: yes
     dest: "{{webui_repo_directory}}"
     version: "{{minemeld_version}}"
-
+- name: git configurating
```

Comment:
Please could you rename the step in something more evocative, like `reconfiguring git to use https`